modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -6014,7 +6014,8 @@ public void onInterruption() {
 
             if (log.isInfoEnabled())
                 log.info("Started serving remote node connection [rmtAddr=" + rmtAddr +
-                    ", rmtPort=" + sock.getPort() + ']');
+                    ", localNodeId=" + locNodeId +
```

Comment:
We could try to detect which node is connecting to this node and print info about remote node.